tests/expander-setup.spec.js

```diff
@@ -0,0 +1,72 @@
+jest.mock('@degjs/dom-event');
+
+import expander from '../src/expander.js';
+
+describe('expander inits', () => {
+    global.console = {
+        log: jest.fn(),
+        error: jest.fn(),
+    };
+
+    afterEach(() => {
+        global.console.log.mockReset();
+        global.console.error.mockReset();
+    });
+
+    it('should handle no containerEl', () => {
+        const expectedMsg = 'Error in expander. No container element provided.';
+        const expanderInst = expander();
+
+        expect(global.console.error).toHaveBeenCalled();
+        expect(global.console.error).toHaveBeenCalledWith(expectedMsg);
+        expect(expanderInst).toEqual({});
+    });
+
+    it('should handle no toggle element available', () => {
+        document.body.innerHTML = `
+            <div data-container></div>
+        `;
+        const containerEl = document.querySelector('[data-container]');
+        const expectedMsg =
+            'Error in expander. No toggle element found in container.';
+        const expanderInst = expander(containerEl);
+
+        expect(global.console.error).toHaveBeenCalled();
+        expect(global.console.error).toHaveBeenCalledWith(expectedMsg);
+        expect(expanderInst).toEqual({});
+    });
+
+    it('should set aria-controls', () => {
```

Comment:
[suggestion] If no `controlsId` setting is passed (which is most use-cases, in practice), then the `aria-controls` attribute should not be set. In this scenario, the accessibility relies on the trigger immediately preceding in the DOM the element that it is controlling. So a possible additional test would be to confirm that the aria attribute isn't getting set.

Reply:
I think I covered quite a few of those in `tests/expander-functionality.spec.js`, but let me know if there are any more that would be good to add